tests/trx2junit.Tests/Internal/JUnitBuilderTests/Build.cs

```diff
@@ -0,0 +1,126 @@
+﻿using NUnit.Framework;
```

Comment:
Nit: move this namespace below the system-namespaces.

Reply:
Coding standards are important 😄  changed both of these